### PR TITLE
Comlete fizz

### PR DESCRIPTION
**Pull request by ulgoon**
resolved: #1

### CODE REVIEW

File: fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+def fizzbuzz(num):
+    for i in range(1,num+1):
+        if i%3==0:
+            print('fizz')
+        else:
+            print(i)
+
+if __name__=='__main__':
+    user_num = int(input('Type number: '))
```
Comment (kingwangzzang1234) [inline fizzbuzz.py:9]: Why did you do that?
move line 9 to line 2

Comment (ulgoon) [reply fizzbuzz.py:9]: Ah.... Okay..

Comment (ulgoon) [reply fizzbuzz.py]: I'm done

**Review by kingwangzzang1234 — APPROVED** [review]
Well done

**Pull request merged**